### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by mwest** [opentimestamps/opentimestamps-client#6]
I have run into 2 problems running on Windows (10).

1) The opentimestamps symlink folder did not work. Deleting and recreating it will fix it...(run as admin) MKLINK /D opentimestamps python-opentimestamps\opentimestamps

2) The SSL certificate is not verifying, possibly due to a known problem when client and server are running different versions of SSL/TLS (https://docs.python.org/2/library/ssl.html#socket-creation). I have temporarily set 'unverifiable=True' in the calendar requests to avoid this.
 
![ots](https://cloud.githubusercontent.com/assets/2612/19015204/1143b1c0-87ce-11e6-9c7b-3f1d0902e64b.png)

**Comment by petertodd** [comment]
1) A way to avoid the symlink is a dynamic path insert: https://github.com/opentimestamps/opentimestamps-client/commit/6af509bb55d65de1b6647ee82d7b70e857abb215 Does that work better for you?

2) Note that this is a Python3 program, not Python2... your link is to Python2 docs. What exact version of Python are you using?

**Comment by petertodd** [comment]
BTW, I dunno if this helps you, but here's the nginx config I'm using on the server:

    proxy_cache_path /tmp/nginx-cache levels=1:2 keys_zone=otsd:10m inactive=3600m max_size=1000m;

    server {
            listen 443 default_server;
            listen [::]:443 default_server;
            server_name btc.opentimestamps.org;

            root /dev/null;

            location /journal {
            alias /var/lib/otsd/calendar/journal;
            }

            location / {
            proxy_cache otsd;
            add_header X-Proxy-Cache $upstream_cache_status always;

            proxy_pass http://127.0.0.1:14788;
            proxy_set_header Host $host;
            }

        ssl on;
        ssl_certificate /etc/letsencrypt/live/pool.opentimestamps.org/fullchain.pem;
        ssl_certificate_key /etc/letsencrypt/live/pool.opentimestamps.org/privkey.pem;

        ssl_session_timeout 5m;

        ssl_ciphers 'ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-DSS-AES128-GCM-SHA256:kEDH+AESGCM:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA:ECDHE-ECDSA-AES256-SHA:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-DSS-AES128-SHA256:DHE-RSA-AES256-SHA256:DHE-DSS-AES256-SHA:DHE-RSA-AES256-SHA:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:AES128-SHA:AES256-SHA:AES:CAMELLIA:DES-CBC3-SHA:!aNULL:!eNULL:!EXPORT:!DES:!RC4:!MD5:!PSK:!aECDH:!EDH-DSS-DES-CBC3-SHA:!EDH-RSA-DES-CBC3-SHA:!KRB5-DES-CBC3-SHA';

        ssl_prefer_server_ciphers on;
        ssl_dhparam /etc/ssl/private/dhparams.pem;
    }

Basically, nginx is being used as a (caching) proxy to the underlying otsd daemon.

**Comment by mwest** [comment]
1) Yes, it works...and it fixed the SSL cert error as well!

2) I am using 3.5.2. (the same underlying problem exists). I don't understand why this got fixed , but since as this is my first experience with python, I'll gladly accept it and move forward!

[i won't close the issue or submit a PR until I get a linux environment setup to also test on, but deleting the symlink file and adding sys.path.insert(1, sys.path[0] + '/python-opentimestamps/') to ots definitely fixed all issues]

**Comment by petertodd** [comment]
Huh, very curious!

@RCasatta Maybe we should add back that sys.path.insert() for windows compatibility? Is there some way it can co-exist with IDE's?

**Comment by RCasatta** [comment]
Yes it's ok to revert 6af509bb55d65de1b6647ee82d7b70e857abb215

I didn't think about symlink not working on windows and I just discover you can mark folder as source in the IDE to solve the problem. 
(Sorry for not applying this before causing the window issue, I didn't know)

<img width="1032" alt="schermata 2016-10-04 alle 09 09 33" src="https://cloud.githubusercontent.com/assets/6470319/19065183/4d8de4e8-8a12-11e6-90c8-ab4cbdf22dde.png">

**Comment by petertodd** [comment]
Done by https://github.com/opentimestamps/opentimestamps-client/commit/ed19fa3da5b5647dbb124d4e48320d34f55e5e5f

@mwest Let me know if this fixes the issue for you.

**Comment by mwest** [comment]
@petertodd Fixed, thanks!
